domain-specific-datasets/README.md

```diff
@@ -0,0 +1,17 @@
+# Domain Specific Dataset Project
+
+This is project to bootstrap the creation of domain-specific datasets for training models. The goal is to create a set of tools that help users to collaborate with domain experts.
```

Comment:
I think a step by step example would be more useful in this case. Even something like a video of you, or anyone else, walking through the idea and steps one-by-one might be even better.